app/src/main/java/ch/epfl/sweng/erpa/model/User.java

```diff
@@ -0,0 +1,31 @@
+package ch.epfl.sweng.erpa.model;
+
+public class User {
+    String username;
```

Comment:
Can I suggest having an authoritarive `userId` and let username be optional: If the user changes his username, we'll have a robust notion of an user. Global (hidden) user uniqueness will avoid problems later.